include/modem/lte_lc.h

```diff
@@ -1166,10 +1491,11 @@ int lte_lc_nw_reg_status_get(enum lte_lc_nw_reg_status *status);
 int lte_lc_system_mode_set(enum lte_lc_system_mode mode,
 			   enum lte_lc_system_mode_preference preference);
 
-/** @brief Get the modem's system mode and LTE preference.
+/**
+ * Get the modem's system mode and LTE preference.
  *
  * @param mode Pointer to system mode variable.
- * @param preference Pointer to system mode preference variable. Can be NULL.
+ * @param preference Pointer to system mode preference variable. Can be @c NULL.
```

Comment:
Remove all "pointer to", "structure to", "structure"

Reply:
I'll rework the function parameters. I'll also add [in]/[out] for all parameters.